sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java

```diff
@@ -469,4 +470,110 @@
         return client.beginRecognizeBusinessCards(toFluxByteBuffer(businessCard), length,
             recognizeBusinessCardsOptions, context).getSyncPoller();
     }
+
+    /**
+     * Recognizes receipt data from document using optical character recognition (OCR) and a prebuilt receipt trained
```

Comment:
this docstring uses receipt instead of invoice

Reply:
🙅